testspec.yml

```diff
@@ -22,6 +22,7 @@ phases:
       - pip install scheduler/.
       - echo Running pytest $TEST_TYPE tests on $DLC_IMAGES...
       - export PYTHONPATH=$PYTHONPATH:$(pwd)/src
+      - export UPGRADE_REPO_NAME=trshanta-repo
```

Comment:
Do we still need it?

Reply:
Yeah, this is needed till we have the temporary infra in place for V1.